modules/dreamview/frontend/src/store/index.js

```diff
@@ -158,10 +158,6 @@ class DreamviewStore {
     }
 
     handleDrivingModeChange(wasAutoMode, isAutoMode) {
-        if (this.options.enableSimControl) {
```

Comment:
We still want to keep this logic. Just this time, we check if SimControl is on from HMI's componentStatus: hmi.componentStatus.get(XXX)

Reply:
I'm having some difficulty here. It seems `this.hmi.componentStatus.has("SimControl")` always returns false.

Reply:
Oops, apologize for pointing to the wrong variable.  This will do the job: `this.hmi.moduleStatus.get("SimControl")`